lang/cs/Org.Apache.REEF.IMRU/API/IIMRUClient.cs

```diff
@@ -32,5 +34,13 @@ public interface IIMRUClient
         /// <param name="jobDefinition">IMRU job definition</param>
         /// <returns>Result of IMRU</returns>
         IEnumerable<TResult> Submit<TMapInput, TMapOutput, TResult>(IMRUJobDefinition jobDefinition);
+
+        /// <summary>
+        /// DriverHttpEndPoint returned by IReefClient after job submission
+        /// returning null is a valid option for implementations that do not run on yarn or multi-core
+        /// For example: see InProcessIMRUCLient.cs
+        /// </summary>
+        [Unstable("0.13", "This depends on IIMRUClient API which itself in unstable ")]
```

Comment:
Nitpick: This depends on an unstable `IREEFClient` API. The reference here is cyclic :-)